### DISCUSSION TITLE
Lineage OS 20

### DISCUSSION THREAD

**Discussion by mival** [agronick/aa-torque#25] [Q&A]
Hi, I have on my OnePlus 6T installed LineageOS 20 (Android 13), with magisk root. I managed to install AA torque as Google Play Store app (KingInstall), but it is not visible in AA (DEV enabled with unknown sources). Have someone any advice?

**Comment by agronick** [comment]
If you have root you might be able to use something like this: https://github.com/Rikj000/Android-Auto-XLauncher-Unlocked

**Reply by mival** [reply]
I tried, but not sure how it should help. I don't still see app in Android auto drawer. It's little weird, because app is listed in Android auto settings on phone
![Screenshot_20231010-203503_Android Auto](https://github.com/agronick/aa-torque/assets/2492366/cf1d7669-98f8-410c-b924-4b02a745c5c1)

**Comment by mival** [comment]
Where I should find the app, in app drawer or is there some special shortcut? Is needed to start headunit server from dev Android auto settings?
